### PR TITLE
M lopez orders controller

### PR DESCRIPTION
**Pull request by naaturaz**
Index View Done. Cart View Started.

### CODE REVIEW

**Comment by ndahlquist-cc** [comment]
+1

File: SeaSharpe_CVGS/SeaSharpe_CVGS/Controllers/OrderController.cs [hunk 1]
```diff
@@ -21,21 +69,20 @@ public class OrderController : Controller
...
-            //    //return ViewEvents view
-            //      return RedirectToAction("OrderHistory");
-            //}
-            //else
-            //{
-            //    //return ViewEvents view
-            //      return RedirectToAction("OrderHistory");
-            //}
+            if (Roles.IsUserInRole(@"employee"))
+            {
```
Comment (BlackthornYugen) [inline SeaSharpe_CVGS/SeaSharpe_CVGS/Controllers/OrderController.cs:73]: Does this still crash here?

File: SeaSharpe_CVGS/SeaSharpe_CVGS/Web.config [hunk 2]
```diff
@@ -22,6 +22,7 @@
     <authentication mode="None" />
     <compilation debug="true" targetFramework="4.5" />
     <httpRuntime targetFramework="4.5" />
+    <roleManager enabled="true" />
```
Comment (BlackthornYugen) [inline SeaSharpe_CVGS/SeaSharpe_CVGS/Web.config:25]: We aren't using role manager

**Comment by ndahlquist-cc** [comment]
It will crash there if you're viewing from the main page.  I'm going to switch those over in all the code they're in once your Branch is merged @BlackthornYugen

File: SeaSharpe_CVGS/SeaSharpe_CVGS/Controllers/OrderController.cs (hunk 1 above)
Comment (naaturaz) [inline SeaSharpe_CVGS/SeaSharpe_CVGS/Controllers/OrderController.cs:73]: Let me knw if I can help

**Comment by BlackthornYugen** [comment]
@ndahlquist-cc I hadn't planned on implementing role manager but after talking with @pjthomso I think it would be a good idea to do so.

**Comment by BlackthornYugen** [comment]
This should be pulled into dan-orders instead of directly to master.

**Pull request closed without merging**